source/collections/VectorList.ooc

```diff
@@ -119,9 +124,12 @@ VectorList: class <T> {
 			result add(this[indices[i]])
 		result
 	}
-	getSlice: func (range: Range) -> This<T> {
-		result := This<T> new(range count)
-		this getSliceInto(range, result&)
+	getSlice: func (start, end: Int) -> This<T> {
```

Comment:
We could still use this signature, though. Keep the old function, but also keep this line and have it call the other, like this:
```ooc
getSlice: func (range: Range) -> This<T> {
	result := This<T> new(range count)
	this getSliceInto(range, result&)
}
getSlice: func (start, end: Int) -> This<T> { this getSlice(start .. end+1) }
```